examples/step-17/step-17.cc

```diff
@@ -148,9 +148,9 @@ namespace Step17
     // parallel versions is denoted the fact that we use the classes from the
     // <code>PETScWrappers::MPI</code> namespace; sequential versions of these
     // classes are in the <code>PETScWrappers</code> namespace, i.e. without
-    // the <code>MPI</code> part). Note also that we do not use a separate
-    // sparsity pattern, since PETSc manages that as part of its matrix data
-    // structures.
+    // the <code>MPI</code> part, be aware that these classes are deprecated).
+    // Note also that we do not use a separate sparsity pattern, since PETSc
```

Comment:
Just remove the whole statement. No need to explain deprecated functionality. We'll forget to remove this sentence when we remove the classes.